helm/ingress-controller/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.6.1
```

Comment:
This release is going to pull in the unreleased crd changes as well right? 
https://github.com/ngrok/kubernetes-ingress-controller/compare/helm-chart-0.6.0...main

Reply:
That is a good point. We might have to manually cut a release for this one unless we are okay with these CRDs going out as is

Reply:
i think at this stage, it would be ok if some of the crds went out as is for now and we release the rest in the next version. It should be soon and its all alpha right now anyways